data/campaigns/Son_Of_The_Black_Eye/scenarios/02_The_Human_Army.cfg

```diff
@@ -21,6 +21,39 @@
             [objective]
                 description= _ "Defeat Lanbec’h"
                 condition=win
+                [show_if]
+                    {VARIABLE_CONDITIONAL prepare_retreat boolean_equals no}
+                    {VARIABLE_CONDITIONAL retreat boolean_equals no}
+                [/show_if]
+            [/objective]
+            [objective]
+#ifdef EASY
+                description= _ "Resist until your people are ready to go (Turn 12)"
+#endif
+#ifdef NORMAL
+                description= _ "Resist until your people are ready to go (Turn 13)"
+#endif
+#ifdef HARD
+                description= _ "Resist until your people are ready to go (Turn 14)"
+#endif
+                condition=win
+                [show_if]
+                    {VARIABLE_CONDITIONAL prepare_retreat boolean_equals yes}
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Then move Kapou’e to the signpost in the Northeast"
+                condition=win
+                [show_if]
+                    {VARIABLE_CONDITIONAL prepare_retreat boolean_equals yes}
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Move Kapou’e to the signpost in the Northeast"
```

Comment:
Shouldn't Northeast be written without capital letter. It's English.

Reply:
So string freeze is over? 
Currently i'm working on LoW too, and there are many dialogues which would need some improvement, and i'd like to add some more.

Reply:
String Freeze is over. It generally ends when a release announcement appears in the forums.
Btw, are you using IRC or Discord?

Reply:
I'm using none of them :( 
Let me know which one you prefer if you have to tell me something. You can send me a private message on wesnoth forum otherwise (i'm Tramontana)

Reply:
With the bridge, it doesn't really matter if you're on IRC or Discord.

Discord allows you to post images, code snippets, and markdown.

IRC clients generally use a lot fewer resources.

I think the main problem people have with IRC is it's so "last millennia"

The problem with both if that you need to be actively monitoring and both parties must be present for a meaningful discussion. But that leads to the argument my wife and I have so often: she likes Facebook and I like e-mail. She expects me to be there when she texts me and to answer immediately. I prefer to wait until I pull off the road and can safely divert my attention.